client/oci_env/utils.py

```diff
@@ -236,6 +236,25 @@ def compose_command(self, cmd, interactive=False, pipe_output=False):
         else:
             return subprocess.run(cmd, capture_output=pipe_output)
 
+    @property
+    def container_name(self):
+        """Docker compose name containers using `-` while podman uses `_`
+
+        docker|podman ps -q --format '{{.Names}}' | grep oci_env | grep pulp
+        """
+        binary = self.config["COMPOSE_BINARY"]
+        cmd = [
+            binary, "ps", "-q",
+            "--filter", f"name={self.config['COMPOSE_PROJECT_NAME']}",
+            "--format", "{{.Names}}",
+        ]
+        running_containers = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+
+        return subprocess.check_output(
```

Comment:
This might break if you have multiple instances of a service running (eg `oci_env-pulp-1`, `oci_env-pulp-2`, `oci_env-pulp-3`)

Reply:
I can grep using a Regexp to get the pattern `^{project}+(_|-)+{service}.*+1$`

Reply:
@rochacbruno that sounds good. Let me know when you get this updated